frontend/public/components/_route.scss

```diff
@@ -10,3 +10,11 @@
 .co-m-route-ingress-status + .co-m-route-ingress-status {
   margin-top: 40px;
 }
+
+.co-m-route-custom-hostname {
+  font-weight: 600;
+}
+
+.co-m-route-custom-help-link {
+  cursor: pointer;
```

Comment:
`cursor: pointer` generally indicates an accessibility problem

Reply:
fixed, although still need a selector to remove the left-padding of a Button